src/components/TodoList/TodoList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from '../Todo/Todo';
+import './TodoList.css';
+
+const TodoList = ({ todos }) => (
+  <ul className="todo__list">
+    {todos.map(todo => (
+      <li key={todo.id} className="todo__item">
+        <Todo todo={todo} />
+      </li>
```

Comment:
```suggestion
      <Todo todoData={currentTodo} key={todo.id} />
```
li перенеси внутрь компонента Todo
И да, три раза подряд слово todo лучше не писать, давай компоненту и props разные названия